### PR TITLE
Add the project's branding banner "dolphin scheduler" at launch

### PR DESCRIPTION
**Pull request by huagetai**
## Add the project's branding banner "dolphin scheduler" at launch
Add the brand banner of the project when the project starts, which will help strengthen the brand awareness of the project.

## Brief change log

  - Add "banner.txt" to src/main/resources
  - Modify src/main/resources/application.yaml

delete  
```
spring:
    main:
        banner-mode: off
```

add
```
spring:
        banner:
            charset: UTF-8
            image:
                width: 20
```

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

**Review by caishunfeng — COMMENTED** [review]
Looks good.

File: dolphinscheduler-standalone-server/src/main/resources/banner.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ _____        _       _     _          _____      _              _       _
+|  __ \      | |     | |   (_)        / ____|    | |            | |     | |
+| |  | | ___ | |_ __ | |__  _ _ __   | (___   ___| |__   ___  __| |_   _| | ___ _ __
+| |  | |/ _ \| | '_ \| '_ \| | '_ \   \___ \ / __| '_ \ / _ \/ _` | | | | |/ _ \ '__|
+| |__| | (_) | | |_) | | | | | | | |  ____) | (__| | | |  __/ (_| | |_| | |  __/ |
+|_____/ \___/|_| .__/|_| |_|_|_| |_| |_____/ \___|_| |_|\___|\__,_|\__,_|_|\___|_|
+               | |
+               |_|
+=====================================================================================
+Dolphinscheduler 版本: ${application.version}
```
Comment (kezhenxu94) [inline dolphinscheduler-standalone-server/src/main/resources/banner.txt:12]: I don't suggest using Chinese words in the terminal, which might have unnecessary encoding issue

Comment (huagetai) [reply dolphinscheduler-standalone-server/src/main/resources/banner.txt:12]: English is better

**Comment by huagetai** [comment]
merger to #9273

**Comment by huagetai** [comment]
@caishunfeng merged to #9273. this close

File: dolphinscheduler-standalone-server/src/main/resources/banner.txt (hunk 1 above)
Comment (caishunfeng) [reply dolphinscheduler-standalone-server/src/main/resources/banner.txt:12]: not resolved

**Comment by caishunfeng** [comment]
> @caishunfeng merged to #9273. this close

**Pull request closed without merging**